core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -136,6 +136,13 @@
         <span translate="I18N_SIDEBAR_CONTACT_US"></span>
       </a>
     </li>
+
+    <li>
+      <a href="http://oppiafoundation.org/" target="_blank">
+        <img ng-src="<[getStaticImageUrl('/sidebar/oppia-foundation-logo.png')]>" class="oppia-sidebar-menu-icon" alt="Oppia Foundation">
```

Comment:
Actually, hang on. Where's the png file being referred to here? I couldn't find it in the assets/images/sidebar folder -- is it coming from somewhere else?

Reply:
oops I forgot to check it in! I'll add it now.

Here's what it'll look like:
<img width="290" alt="oppia-foundation" src="https://user-images.githubusercontent.com/11008603/42981688-0f169c0e-8bfb-11e8-9106-68431c635794.png">